app/src/main/java/com/peacecorps/pcsa/support_services/StepsFragment.java

```diff
@@ -0,0 +1,45 @@
+package com.peacecorps.pcsa.support_services;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.AppCompatActivity;
+import android.text.Html;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.R;
+
+/**
+ * Steps in Reporting
+ *
+ * @author Buddhiprabha Erabadda
```

Comment:
Yes, @Buddhiprabha is right
This layout had been previously created by Buddhiprabha. I have only modified it a bit in the XML to suit the new mockups. Also, in the java file, I have made only the required changes to make it a Fragment. 

Hence, I copied the comment of Steps class. 
Should I make changes to this?